descriptors/ModuleDescriptor-template.json

```diff
@@ -305,7 +301,7 @@
       },
       {
         "name": "ENV",
-        "value": "folio"
+        "value": "FOLIO"
       }
```

Comment:
For correct kafka configuration or it is not needed, Please correct me if it is wrong

Reply:
Ok, Thanks